src/templates/React InstantSearch widget/src/lib/connector.ts.hbs

```diff
@@ -1,13 +1,27 @@
+import type {
+  ConnectedComponentClass,
+  ConnectorProvided,
+} from 'react-instantsearch-core';
+
 import { createConnector } from 'react-instantsearch-dom';
 
+type Connector<TProvided = {}, TExposed = {}> = ((
+  stateless: React.FunctionComponent<ConnectorProvided<TProvided>>
+) => React.ComponentClass<TExposed>) &
+  (<TProps extends Partial<ConnectorProvided<TProvided>>>(
+    Composed: React.ComponentType<TProps>
+  ) => ConnectedComponentClass<TProps, ConnectorProvided<TProvided>, TExposed>);
```

Comment:
The only way I could find to fix a TypeScript error during `build:types` is to explicitly set the type (by copying it from **@types/react-instantsearch-core**) and assign it to the connector.

Here's the issue for reference:

```
$ tsc -p tsconfig.declaration.json
src/lib/connector.ts:19:14 - error TS2742: The inferred type of 'connectWidget' cannot be named without a reference to 'react-instantsearch-core/node_modules/@types/react'. This is likely not portable. A type annotation is necessary.

19 export const connectWidget = createConnector<ProvidedProps>({
                ~~~~~~~~~~~~~
```